changelog.d/+preview-env.added.md

```
@@ -0,0 +1 @@
+mirrord now supports "preview environments" - a new type of mirrord session that lives directly in the cluster and can be shared with other users. Read more about it [here](PLACEHOLDER-THERE-ARE-NO-DOCS-YET).
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)